microprofile/health-smallrye/src/main/java/org/wildfly/extension/microprofile/health/MicroProfileHealthReporter.java

```diff
@@ -166,7 +166,13 @@ private SmallRyeHealth getHealth(Map<HealthCheck, ClassLoader> serverChecks, Map
         builder.add("status", status.toString());
         builder.add("checks", checkResults);
 
-        return new SmallRyeHealth(builder.build());
+        JsonObject build = builder.build();
+
+        if (status.equals(HealthCheckResponse.Status.DOWN)) {
+            MicroProfileHealthLogger.LOGGER.healthDownStatus(build.toString());
```

Comment:
Is this a one off log or does this get logged periodically whilst down?

Reply:
every time the health reports DOWN